pkg/apis/pravega/v1alpha1/bookkeeper.go

```diff
@@ -11,18 +11,137 @@
 package v1alpha1
 
 import (
+	"fmt"
+
 	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+const (
+	// DefaultBookkeeperImageRepository is the default Docker repository for
+	// the BookKeeper image
+	DefaultBookkeeperImageRepository = "pravega/bookkeeper"
+
+	// DefaultBookkeeperImageTag is the default tag used for for the BookKeeper
+	// Docker image
+	DefaultBookkeeperImageTag = "latest"
+
+	// DefaultBookkeeperImagePullPolicy is the default image pull policy used
+	// for the BookKeeper Docker image
+	DefaultBookkeeperImagePullPolicy = v1.PullAlways
+
+	// DefaultBookkeeperLedgerVolumeSize is the default volume size for the
+	// Bookkeeper ledger volume
+	DefaultBookkeeperLedgerVolumeSize = "10Gi"
+
+	// DefaultBookkeeperJournalVolumeSize is the default volume size for the
+	// Bookkeeper journal volume
+	DefaultBookkeeperJournalVolumeSize = "10Gi"
+
+	// MinimumBookkeeperReplicas is the minimum number of Bookkeeper replicas
+	// accepted
+	MinimumBookkeeperReplicas = 3
```

Comment:
The other constants are prefixed with `Default`, why is this one not prefixed?

Reply:
In this particular case, I thought that `Minimum` better represents the purpose of this constant.

Reply:
Ok, this value is playing two roles then: default and minimum.